src/app/Library/CrudPanel/Traits/FieldsProtectedMethods.php

```diff
@@ -18,7 +18,6 @@ trait FieldsProtectedMethods
     protected function makeSureFieldHasNecessaryAttributes($field)
     {
         $field = $this->makeSureFieldHasName($field);
-        $field = $this->makeSureFieldHasModel($field);
```

Comment:
Hey @tabacitu 

I removed it because if field has entity (relation) we get the model from the relation, in my view there is no sense if field has entity to get the model from other place than the relation.